src/wolfsftp.c

```diff
@@ -38,6 +38,17 @@
 #endif
 
 
+#ifndef SIZEOF_OFF_T
```

Comment:
If you include port.h in wolfsftp.c, you won't need this block here.

Reply:
updated